components/ide/jetbrains/image/gha-update-image/index.js

```diff
@@ -12,56 +12,56 @@ const IDEs = [
         productId: "intellij",
         productCode: "IIU",
         productType: "release",
-        exampleRepo: "https://github.com/gitpod-io/spring-petclinic",
+        exampleRepo: "https://github.com/gitpod-samples/spring-petclinic",
     },
     {
         productName: "GoLand",
         productId: "goland",
         productCode: "GO",
         productType: "release",
-        exampleRepo: "https://github.com/gitpod-io/template-golang-cli",
+        exampleRepo: "https://github.com/gitpod-samples/template-golang-cli",
     },
     {
         productName: "PyCharm Professional Edition",
         productId: "pycharm",
         productCode: "PCP",
         productType: "release",
-        exampleRepo: "https://github.com/gitpod-io/template-python-django",
+        exampleRepo: "https://github.com/gitpod-samples/template-python-django",
     },
     {
         productName: "PhpStorm",
         productId: "phpstorm",
         productCode: "PS",
         productType: "release",
-        exampleRepo: "https://github.com/gitpod-io/template-php-laravel-mysql",
+        exampleRepo: "https://github.com/gitpod-samples/template-php-laravel-mysql",
     },
     {
         productName: "RubyMine",
         productId: "rubymine",
         productCode: "RM",
         productType: "release",
-        exampleRepo: "https://github.com/gitpod-io/template-jetbrains-rubymine",
+        exampleRepo: "https://github.com/gitpod-samples/template-jetbrains-rubymine",
     },
     {
         productName: "WebStorm",
         productId: "webstorm",
         productCode: "WS",
         productType: "release",
-        exampleRepo: "https://github.com/gitpod-io/template-jetbrains-webstorm",
+        exampleRepo: "https://github.com/gitpod-samples/template-jetbrains-webstorm",
```

Comment:
These two repositories currently doesn't exist:
- https://github.com/gitpod-samples/template-jetbrains-rubymine
- https://github.com/gitpod-samples/template-jetbrains-webstorm

Reply:
@felladrin I am not sure what we should do. Since, old repos. are also deprecated 😢

Reply:
Resolved in [`f676b18`](https://github.com/gitpod-io/gitpod/pull/15502/commits/f676b1829246c6cbb20ca7843e956be909ee0624)

Thanks @felladrin for the suggestions ❤️